src/pages/view-test-result/__tests__/view-test-result.spec.ts

```diff
@@ -116,6 +119,25 @@ describe('ViewTestResultPage', () => {
         expect(result).toBeNull();
       });
     });
+    describe('getTestSummary', () => {
+      it('should return the correct data', () => {
+        component.testResult = categoryBTestResultMock;
+        const result: TestSummaryCardModel = component.getTestSummary();
+        expect(result.D255).toBeFalsy();
+        expect(result.accompaniment).toEqual(['ADI', 'Interpreter']);
+        expect(result.candidateDescription).toBe('mock-candidate-description');
+        expect(result.debriefWitnessed).toBeTruthy();
+        expect(result.independentDriving).toBe('Sat nav');
+        expect(result.passCertificateNumber).toBe('mock-pass-cert-number');
+        expect(result.provisionalLicenceProvided).toBeTruthy();
+        expect(result.routeNumber).toBe(12345);
```

Comment:
Can we use `toEqual(true)` and `toEqual(false)` instead of `toBeTruthy()` and `toBeFalsy()`?